test/output_test.py

```diff
@@ -10,36 +12,71 @@ def isatty(self):
         return True
 
 
-def test_step():
+def test_success():
     f = TTY()
-    o = output.TextOutput(f)
-    o.step("Step")
-    assert f.getvalue() == "Step\n"
+    o = output.TextOutput(4, file=f)
+    o.step("Getting change %s info", "54321")
+    o.step("Starting system tests %s suite", "basic")
+    o.step("Waiting for job %s", "https://jenkins.ovirt.org/job/98765")
+    o.success("Job completed successfully!")
 
+    out = f.getvalue()
+    assert out == """\
+[ 1/4 ] Getting change 54321 info
+[ 2/4 ] Starting system tests basic suite
+[ 3/4 ] Waiting for job https://jenkins.ovirt.org/job/98765
+[ 4/4 ] {}Job completed successfully!{}
+""".format(output.GREEN, output.RESET)
 
-def test_success():
-    f = TTY()
-    o = output.TextOutput(f)
-    o.success("msg")
-    assert f.getvalue() == output.GREEN + "msg" + output.RESET + "\n"
+    print()
```

Comment:
what does this do?

Reply:
This allow insepecting the output by running the tests with -s:

    tox -- -s